src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder saveDataToStringFromExistFile = new StringBuilder();
+
+        try {
+            BufferedReader readerFromExistFile = new BufferedReader(new FileReader(fromFileName));
+            String value = readerFromExistFile.readLine();
+
+            while (value != null) {
+                saveDataToStringFromExistFile.append(value).append(" ");
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
                saveDataToStringFromExistFile.append(value).append(SEPARATOR);
```